src/test/java/posters/tests/smoke/BrowseRandomVisualAssertTest.java

```diff
@@ -61,21 +57,20 @@ public void setup()
     @Test
     public void browseRandomCategoriesAndProducts()
     {
-        CategoryPage categoryPage;
-        ProductdetailPage productPage;
+        var categoryPage = new CategoryPage();
```

Comment:
Please revert to: 
```
CategoryPage categoryPage;
ProductdetailPage productPage;
```
The initialization takes more characters and I would not initiate so that a NullpointerException would occur since the page was not opened at this time.